tests/foreman/api/test_permission.py

```diff
@@ -205,14 +200,13 @@ def give_user_permission(self, perm_name):
         self.user = self.user.update(['role'])
 
     @tier1
-    def test_create(self):
+    def test_positive_create_permission(self):
```

Comment:
Our test module has name `test_permission`, that is why I put my suggestion
Maybe, `test_check_create`